docs/installing_with_spack.md

```diff
@@ -36,7 +36,7 @@ source /path-to-spack/share/spack/setup-env.sh
 spack compiler find
 spack install exago@develop%gcc \
   ^openmpi ^ipopt@3.12.10+coinhsl~mumps ^coinhsl+blas \
-  ^petsc@3.13.6+mpi~hypre~superlu-dist~mumps+shared
+  ^petsc@3.19+mpi~hypre~superlu-dist~mumps+shared
```

Comment:
```suggestion
  ^petsc+mpi~hypre~superlu-dist~mumps+shared
```

Would this be more accurate perhaps? We should merge this

Reply:
Sounds good!